content/en/tracing/error_tracking/_index.md

```diff
@@ -1,39 +1,50 @@
-- __Following issues over time__ to show when they first started, if they are ongoing, and how often they are occurring, helping you identify the most important ones.
-- __Getting all the context in one place__ to facilitate troubleshooting.
+## Use span tags to track error spans
 
-## How Datadog error tracking works
+<div class="alert alert-info">Error Tracking is available for all the languages supported by APM and does not require using a different SDK.</div>
 
-The Datadog tracers collect errors through integrations and manual instrumentation of the source code. Error spans within a trace are processed by Error Tracking __when they are located in the uppermost service span__, which is also called the _service entry span_. The span must also contain the `error.stack`, `error.msg`, and `error.type` [span tags][1] in order to be tracked.
+The Datadog tracers collect errors through integrations and the manual instrumentation of your backend services' source code. If an error span within a trace is processed by Error Tracking **when it is located in the uppermost service span**, it is called a _service entry span_. This span must contain the `error.stack`, `error.message`, and `error.fingerprint` [span tags][1] to be tracked.
 
-{{< img src="tracing/error_tracking/flamegraph_with_errors.png" alt="Flame graph with errors"  >}}
+{{< img src="tracing/error_tracking/flamegraph_with_errors.png" alt="Flame graph with errors" style="width:90%;" >}}
 
-Error Tracking computes a fingerprint for each error span it processes using the error type, the error message, and the frames that form the stack trace. Errors with the same fingerprint are grouped together and belong to the same _issue_.
+Error Tracking computes a fingerprint for each error span it processes using the error type, the error message, and the frames that form the stack trace. Errors with the same fingerprint are grouped together and belong to the same issue. For more information, see the [Trace Explorer documentation][2].
 
-<div class="alert alert-info">Error Tracking is available for all the languages supported by APM and does not require using a different SDK.</div>
+## Examine issues to start troubleshooting or debugging
+
+Error Tracking automatically categorizes errors into issues collected from your backend services in the [Error Tracking Explorer][1]. 
+
+Click on an issue to see a summary of the error, the distribution of impacted spans, the latest most relevant stack trace, span tags, host tags, container tags, and metrics.
+
+Error tracking enables you to:
+- Set alerts on Error Tracking events. This helps you to remain informed of fatal issues that may occur.
+- Group similar errors into issues, so that you can more easily identify important errors and reduce noise.
+- Follow issues over time to know when they first started, if they are still ongoing, and how often they are occurring.
+- Collect all the necessary context in one place to facilitate troubleshooting.
```

Comment:
This list feels like overview, and should go back at the top where it was, not after the instructions telling you how those things are done (grouping into issues, for example). Probably true in the RUM page too.